pkg/jobs/canary/canary_jobs.go

```diff
@@ -111,6 +112,11 @@ func (job CanaryJob) Run() {
 		logger.Errorf("error deleting transformed checks for canary %s: %v", job.Canary.GetPersistedID(), err)
 	}
 
+	if len(checksToMarkHealthy) > 0 {
+		// Remove from cron and let the transformed checks repopulate
```

Comment:
Not sure I understand this- Why do we remove the schedule?

Reply:
When a transformed check is marked healthy, it is technically soft deleted but it still remains in the cronjob. We delete the cronentry and the SyncCanaryJob repopulates the cron with non deleted checks